### ISSUE TITLE
CodeBlock.builder() is missing addComment() method

### ISSUE THREAD

**Issue by HoffiMuc** [square/kotlinpoet#1690]
Why does CodeBlock.builder() not have addComment() method like e.g. FunSpec.builder() ?

**Comment by Egorand** [comment]
What would be a use-case for it? A comment _is_ a CodeBlock.

**Comment by HoffiMuc** [comment]
I am forced to do (as I am in a sub-function that only has a CodeBlock.builder() instance and not its FunSpec.builder()
```
bodyBuilder.addStatement("// not yet implemented ${prop.name()} PoetType of %T", prop.poetType)
```

as there just is missing a
```
bodyBuilder.addComment("not yet implemented ${prop.name()} PoetType of %T", prop.poetType)
```

but with addStatement("// ...")
if the line is too long and a line break happens, the generated code obviously doesn't compile anymore.

**Comment by Egorand** [comment]
Yeah, I think that makes sense. I originally felt like "comment" is a concept on a higher level than "CodeBlock", and there are many things we represent with CodeBlocks which don't support comments, but that said, we do have API like `beginControlFlow()`, which definitely can't be used everywhere a CodeBlock can. Do you want to send a PR?

Meanwhile you can copy what `FunSpec.Builder.addComment()` does to ensure there are no line breaks inside of a comment:

```kotlin
public fun addComment(format: String, vararg args: Any): Builder = apply {
  body.add("//·${format.replace(' ', '·')}\n", *args)
}
```
